src/middlewares/cors.js

```diff
@@ -1,15 +1,26 @@
 const defaults = {
-  origin: '*',
+  origin: '*', // Can also be an array of whitelisted origins
```

Comment:
Maybe leave `origin` property as it is, and introduce a new property `origins`? To not mix different data types in 1 property and avoid if/else checks for data types?

Reply:
Pushed changes to add origins